src/scenes/AcceleratorRouter/FundingEntities/EditView.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Page from 'src/components/Page';
+import { APP_PATHS } from 'src/constants';
+import { useFundingEntity } from 'src/providers';
+import { FundingEntity } from 'src/types/FundingEntity';
+import useStateLocation, { getLocation } from 'src/utils/useStateLocation';
+
+import FundingEntityForm from './FundingEntityForm';
+import useUpdateFundingEntity from './useUpdateFundingEntity';
+
+const EditView = () => {
+  const navigate = useNavigate();
+  const location = useStateLocation();
+  const { fundingEntity, reload } = useFundingEntity();
+  const updateFundingEntity = useUpdateFundingEntity();
+
+  const goToViewFundingEntity = useCallback(() => {
+    reload();
+    navigate(
+      getLocation(
+        APP_PATHS.ACCELERATOR_FUNDING_ENTITIES_VIEW.replace(':fundingEntityId', `${fundingEntity?.id}`),
+        location
+      )
+    );
+  }, [navigate, location, fundingEntity]);
+
+  const handleOnSave = useCallback(
+    (record: FundingEntity) => {
+      updateFundingEntity.update(record);
+    },
```

Comment:
Should go to view funding entity after update?

Reply:
Yes, lines 36-40 handle that depending on if the update succeeded or not. This was basically a copy of `People/EditView.tsx` but if there's a different way that we prefer for this I'm good to change it

Reply:
Oh, no, that's fine! I just missed that useEffect